src/main/kotlin/com/terraformation/backend/seedbank/model/Accession.kt

```diff
@@ -161,6 +145,24 @@ data class AccessionModel(
   val active: AccessionActive?
     get() = state?.toActiveEnum()
 
+  fun toV1Compatible(clock: Clock): AccessionModel {
+    return if (!isManualState) {
+      this
+    } else {
+      copy(isManualState = false).withCalculatedValues(clock)
+    }
+  }
+
+  fun toV2Compatible(clock: Clock): AccessionModel {
+    return if (isManualState) {
+      this
+    } else {
+      withCalculatedValues(clock)
```

Comment:
is this first withCalculatedValues intentional?

Reply:
Yeah, but good point that it's not obvious why. Added a comment explaining what it's for.

Reply:
Thanks! Makes sense, had a rough idea.